.github/workflows/main.yml

```diff
@@ -0,0 +1,80 @@
+# Source: https://github.com/pharmaverse/admiralci
+# Stripped down version of admiralci checks. Good for
+# the developement process. When package is ready to
+# be published, revisit and add release-related
```

Comment:
suggestion: might be good to list every missing workflow/job that is release-related. This way it will be easier to come back to it later and just copy-paste necessary stuff.